vitest.workspace.ts

```diff
@@ -0,0 +1,27 @@
+import {defineWorkspace} from 'vitest/config';
+
+export default defineWorkspace([
+  {
+    test: {
+      name: 'Node.js',
+      environment: 'node',
+      restoreMocks: true,
+    },
+  },
+
+  // TODO: enable for browser tests
```

Comment:
Note: I was able to get the browser tests set up locally, but it (correctly) errs on

```
module "browserslistToTargets.js" declares no export named "browserslistToTargets"
```

as it’s CJS, attempting to run as an ESM import. I don’t want to touch any runtime with a 10' pole, especially not in this PR, so maybe there’s a way we could build this file or something just to get the proper behavior in tests.